### PR TITLE
[AIRFLOW-6939] Executor configuration via import path

### PR DESCRIPTION
**Pull request by mik-laj**
I want to use it with MockExecutor. MockExecutor is helpful for performance tests.

---
Issue link: [AIRFLOW-6939](https://issues.apache.org/jira/browse/AIRFLOW-6939)

Make sure to mark the boxes below before creating PR: [x]

- [X] Description above provides context of the change
- [X] Commit message/PR title starts with `[AIRFLOW-NNNN]`. AIRFLOW-NNNN = JIRA ID<sup>*</sup>
- [X] Unit tests coverage for changes (not needed for documentation changes)
- [X] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [X] Relevant documentation is updated including usage instructions.
- [X] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

<sup>*</sup> For document-only changes commit message can start with `[AIRFLOW-XXXX]`.

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

File: UPDATING.md [hunk 1]
```diff
@@ -61,6 +61,29 @@ https://developers.google.com/style/inclusive-documentation
...
+custom executor, but you need to provide the full path to the module in the `executor` option
+in the `core` section. The purpose of this change is to simplify the plugin mechanism and make
+it easier to configure executor.
+
+If your module was in the path `my_acme_company.executors.MyCustomExecutor`  and the plugin was
+called `my_plugin` then your configuration looks like this
+
+```ini
+[core]
+executor = my_plguin.MyCustomExecutor
```
Comment (ashb) [inline UPDATING.md:77]: Did this even work before? I thought we only looked at a specific hard-coded list?

Comment (ashb) [reply UPDATING.md:77]: More importantly for updating -- did we ever publish executors-via- plugin to 1.10 release, if not then these updating instructions don't make sense.

Comment (mik-laj) [reply UPDATING.md:77]: II haven't tested it, but the code looked like it had a chance to work.  In the documentation, we have information:
https://airflow.readthedocs.io/en/1.10.6/plugins.html

Comment (ashb) [reply UPDATING.md:77]: Oh wow missed that one totally!

**Review by turbaszek — APPROVED** [review]

**Pull request merged**